substrate/bin/node/testing/src/keyring.rs

```diff
@@ -20,52 +20,61 @@
 
 use codec::Encode;
 use kitchensink_runtime::{CheckedExtrinsic, SessionKeys, SignedExtra, UncheckedExtrinsic};
-use node_cli::chain_spec::get_from_seed;
 use node_primitives::{AccountId, Balance, Nonce};
-use sp_core::{ecdsa, ed25519, sr25519};
+use sp_core::{ecdsa, ed25519, sr25519, Pair};
 use sp_crypto_hashing::blake2_256;
-use sp_keyring::AccountKeyring;
+use sp_keyring::Sr25519Keyring;
 use sp_runtime::generic::Era;
 
 /// Alice's account id.
 pub fn alice() -> AccountId {
-	AccountKeyring::Alice.into()
+	Sr25519Keyring::Alice.into()
 }
 
 /// Bob's account id.
 pub fn bob() -> AccountId {
-	AccountKeyring::Bob.into()
+	Sr25519Keyring::Bob.into()
 }
 
 /// Charlie's account id.
 pub fn charlie() -> AccountId {
-	AccountKeyring::Charlie.into()
+	Sr25519Keyring::Charlie.into()
 }
 
 /// Dave's account id.
 pub fn dave() -> AccountId {
-	AccountKeyring::Dave.into()
+	Sr25519Keyring::Dave.into()
 }
 
 /// Eve's account id.
 pub fn eve() -> AccountId {
-	AccountKeyring::Eve.into()
+	Sr25519Keyring::Eve.into()
 }
 
 /// Ferdie's account id.
 pub fn ferdie() -> AccountId {
-	AccountKeyring::Ferdie.into()
+	Sr25519Keyring::Ferdie.into()
 }
 
 /// Convert keyrings into `SessionKeys`.
 pub fn session_keys_from_seed(seed: &str) -> SessionKeys {
+	// use std::str::FromStr;
+	let sr25519_seed = sr25519::Pair::from_string(&format!("//{}", seed), None)
+		.expect("should parse str seed to sr25519 public")
+		.public();
 	SessionKeys {
-		grandpa: get_from_seed::<ed25519::Public>(seed).into(),
-		babe: get_from_seed::<sr25519::Public>(seed).into(),
-		im_online: get_from_seed::<sr25519::Public>(seed).into(),
-		authority_discovery: get_from_seed::<sr25519::Public>(seed).into(),
-		mixnet: get_from_seed::<sr25519::Public>(seed).into(),
-		beefy: get_from_seed::<ecdsa::Public>(seed).into(),
+		grandpa: ed25519::Pair::from_string(&format!("//{}", seed), None)
+			.expect("should parse str seed to sr25519 public")
+			.public()
+			.into(),
```

Comment:
Can we add a `from_seed` method on the `Pair` struct, that does this thing? Make it more compact and guides people through how to convert a string to a `Pair` (as you see, it is not enough to use just the `seed` string).

Reply:
> Can we add a `from_seed` method on the `Pair` struct, that does this thing? Makes it more compact and guides people through how to convert a string to a `Pair` (as you see, it is not enough to use just the `seed` string).
> 
> LE: there are multiple places in the code which will benefit from a compact version too (I think @michalkucharczyk commented on the same lines too).

Good idea. I always try to avoid keep `get_from_seed` in the same place.